### PR TITLE
Update service-template-apps-create.md

### PR DESCRIPTION
**Pull request by marclelijveld**
Added note that feature switch for tempalte apps is an irreversible operation.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@paulinbar 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by paulinbar** [comment]
@teddyberco, @amirshwar, can one of you review and accept/reject/modify the @marclelijveld's suggested change? Thanks!

File: powerbi-docs/connect-data/service-template-apps-create.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ To create a template app you can distribute to other Power BI tenants, you need
    >[!Important]
    > The capacity that the app builder workspace is assigned to does not determine the capacity assignment of workspaces where app installers install the app. This means that an app developed in a premium capacity workspace will not necessarily be installed on a premium capacity workspace. Therefore it is **not** recommended to use premium capacity for the builder workspace, as installer workspaces might not be premium capacity, and functionality that relies on premium capacity won't work unless the installer manually reassigns the installed workspace to premium capacity.
 
+   >[!Important]
+   > Enabling the feature switch to develop template apps in the workspace settings, is an irreversible operation.
```
Comment (amirshwar) [inline powerbi-docs/connect-data/service-template-apps-create.md:42]: I'd explain it like that:
Enabling the "Develop template app" switch can only be done when the app builder workspace is created and cannot changed later from workspace settings.

**Review by paulinbar — APPROVED** [review]

**Comment by paulinbar** [comment]
#sign-off

**Pull request merged**